src/main/resources/db/migration/V041__Update_document_uuid_values.sql

```diff
@@ -0,0 +1,3 @@
+-- Extracts the text after the last slash (/) in the documentUrl
+UPDATE scannable_items
+ SET documentUuid = REVERSE(SPLIT_PART(REVERSE(documentUrl), '/', 1))
```

Comment:
are both slots writing to that new column now?

Reply:
the new column is in both slots. The previous pr (which writes the value to the new column) is merged but deployed once. the second build  is still in progress

Reply:
should we include a where clause here where documentuuid is null ?

Reply:
added where clause